portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,27 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    // public static void main(String[] args){ 
+    // ArrayList<String> favorite = new ArrayList<String>();
+    //     favorite.add("Red");
+    //     favorite.add("Orange");
+    //     favorite.add("Pho");
+        
+    // }
+ 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Rashad!</h1>");
+     ArrayList<String> favorite = new ArrayList<String>();
+        favorite.add("Red");
+        favorite.add("Orange");
+        favorite.add("Pho");
+        String color = favorite.get(0);
```

Comment:
Color, fruit and food are not used. Do you have some thoughts on how to use them in the future code changes?